Havit.Blazor.Components.Web.Bootstrap/wwwroot/HxSearchBox.js

```diff
@@ -0,0 +1,13 @@
+﻿export function preventDefaultOnKeyDownOnKeys(inputId, hxSearchBoxDotnetObjectReference, keys) {
```

Comment:
Please follow the pattern from other components and make this `initialize()` function.